### PR TITLE
Ajouter l'AAH 

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to sgmap/mes-aides-ui#172

### CODE REVIEW

File: lib/config/ludwig.js [hunk 1]
```diff
@@ -144,5 +144,11 @@ module.exports = [
         'imgSrc':'logo_mairie_paris.png',
         'link':'http://www.paris.fr/pratique/toutes-les-aides-et-allocations/aides-sociales/paris-logement-familles-prestation-ville-de-paris/rub_9737_stand_88805_port_24193',
         'isBaseRessourcesYearMoins2':false
+    },
+    {
+        'id': 'aah',
+        'shortLabel':'AAH',
+        'label': 'Allocation aux adultes handicapés',
+        'isMontantAnnuel': false
```
Comment (MattiSG) [inline lib/config/ludwig.js]: Are you sure this mention is mandatory?

File: lib/models/situation.js [hunk 2]
```diff
@@ -150,7 +150,7 @@ var IndividuDef = {
     interruptedRessources: [{ type: String, enum: ressourceTypes }],
     autresRevenusTnsActiviteType: { type: String, enum: ['achat_revente', 'bic', 'bnc'] },
     autresRevenusTnsEmployes: Boolean,
-    tauxInvalidite: { type: String, enum: ['nul', 'moins50', 'moins80', 'plus80'] },
+    tauxIncapacite: { type: String, enum: ['nul', 'moins50', 'moins80', 'plus80'] },
```
Comment (MattiSG) [inline lib/models/situation.js]: :+1:

File: lib/simulation/openfisca/mapping/index.js [hunk 3]
```diff
@@ -119,18 +119,23 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
     });
 };
 
-var setNonInjectedPrestationsToZero = function(openFiscaEntity, dateDeValeur) {
+var setNonInjectedPrestationsToZero = function(famille, individus, dateDeValeur) {
```
Comment (MattiSG) [inline lib/simulation/openfisca/mapping/index.js]: :+1:

File: lib/simulation/openfisca/mapping/index.js [hunk 4]
```diff
@@ -119,18 +119,23 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
...
     var periods = getPeriods(dateDeValeur);
     var targetPeriods = periods.last12Months.concat([periods.rollingYear]);
-
+    var prestations = _.pick(PRESTATIONS, function(definition) {
```
Comment (MattiSG) [inline lib/simulation/openfisca/mapping/index.js]: Indeed, good performance improvement.

But the varname is ununderstandable: the prestations are the prestations that are of type `Number`?!

File: lib/simulation/openfisca/mapping/reverse.js [hunk 5]
```diff
@@ -12,8 +12,8 @@ module.exports = function reverseMap(openFiscaResponse, situation) {
     })));
     var injectedPrestations = _.intersection(_.keys(PRESTATIONS), injectedRessources);
 
-    // Openfisca famille object, which contains the values of computed prestations
-    var openFiscaFamille = openFiscaResponse.value[0].familles[0];
+    // Computed prestations can be in the 'familles' or 'individus' openfisca entity
+    var openfiscaResults = _.merge(openFiscaResponse.value[0].familles[0], openFiscaResponse.value[0].individus[0]);
```
Comment (MattiSG) [inline lib/simulation/openfisca/mapping/reverse.js]: Keep consistent casing of `openFisca` (I agree with it being a single word in this context and thus all lowercase, but the rest of the code considers it as two words).

**Comment by MattiSG** [comment]
GTM

File: lib/simulation/openfisca/mapping/index.js (hunk 4 above)
Comment (fpagnoux) [inline lib/simulation/openfisca/mapping/index.js]: I guess we need a domain term to depict this kind of prestations (where the benefit is an amount of money).
Prestation monétaire ? allocation ?

Comment (fpagnoux) [inline lib/simulation/openfisca/mapping/index.js]: Financière ?

Comment (fpagnoux) [inline lib/simulation/openfisca/mapping/index.js]: I going for financière after a quick open space brainstorm :)

**Comment by fpagnoux** [comment]
Still GTM ?

**Pull request merged**